src/api/app/models/event_subscription/find_for_event.rb

```diff
@@ -6,34 +6,89 @@ def initialize(event)
       @event = event
     end
 
-    def subscriptions(channel = :instant_email)
+    def subscriptions(channel = :instant_email, debug: false)
+      if debug
+        # The logger output spoils the debug output. Therefore its silenced when running in debug mode.
```

Comment:
This is only true in development mode in which you probably will never run this :)

Reply:
I removed the part where I silence the active record logger